composer.json

```diff
@@ -15,10 +15,10 @@
   },
   "require-dev": {
     "doctrine/coding-standard": "^6.0",
-    "phpbench/phpbench": "^0.14.0",
-    "phpstan/phpstan": "0.11.16",
-    "phpstan/phpstan-phpunit": "0.11.2",
-    "phpstan/phpstan-strict-rules": "0.11.1",
+    "phpbench/phpbench": "^0.14",
+    "phpstan/phpstan": "^0.12",
+    "phpstan/phpstan-phpunit": "^0.12",
+    "phpstan/phpstan-strict-rules": "^0.12",
```

Comment:
We need to lock patch versions as well. New "patch" release might break build otherwise.

Reply:
`0.12.0`